data/en/pipelines.json

```diff
@@ -0,0 +1,92 @@
+{
+  "why": [
+    {
+      "title": "Pipelines benefits",
+      "description": "Pipelines accelerator enables developers, business analysts and data scientists to quickly derive insights from data, without having to worry about infrastructure and integration.",
+      "items": [
+        {
+          "header": "Integrated with any and all data",
+          "paragraph": "Pipelines offers a rich ecosystem of connectors for a variety of legacy systems, relational databases, file systems, cloud services, object stores, NoSQL, EBCDIC, and more.",
+          "id": "reduced-complexity"
+        },
+        {
+          "header": "Increased </br>flexibility",
+          "paragraph": "Pipelines allows portability across on-premises and public cloud environments",
+          "id": "increased-velocity"
+        },
+        {
+          "header": "Reduced complexity, increased productivity",
+          "paragraph": "Pipelines has a visual interface for building data pipelines, code free transformations, and reusable pipeline blueprints.",
+          "id": "increased-flexibility"
+        },
+        {
+          "header": "Improved data trustworthiness",
+          "paragraph": "Pipelines offers data quality libraries, metadata and lineage capture, audit logging.",
+          "id": "improved-visibility"
+        }
+      ]
+    }
+  ],
+  "features_pipeline": [
+    {
+      "title": "Pipeline features",
```

Comment:
somehow, preparation features show up for pipelines:
![image](https://user-images.githubusercontent.com/3895099/55760506-2a2c4900-5a11-11e9-9dea-12077e1d51d4.png)

Reply:
done. Please recheck.

Reply:
Done. Please recheck